ckan/lib/helpers.py

```diff
@@ -2832,3 +2832,23 @@ def make_login_url(
 @core_helper
 def csrf_input():
     return snippet('snippets/csrf_input.html')
+
+
+@core_helper
+def get_null_label() -> str:
+    _auto_flask_context = _get_auto_flask_context()
+
+    if _auto_flask_context:
+        _auto_flask_context.push()
+
+    null_label = config.get('ckan.null_label', None)
+    if null_label is not None:
+        try:
+            null_label = json.loads(null_label)
+            if lang() in null_label:
+                return null_label[lang()]
```

Comment:
this can fail with a KeyError. Maybe supporting translated text in the default helper is more complexity than we need for this feature.

Reply:
No longer supporting json string for the option. Users can translate in po/mo files if they want. (put the option value through `toolkit._` now).